docs/labels.md

```diff
@@ -84,6 +84,7 @@ larger set of contributors to apply/remove them.
 | <a id="triage/needs-information" href="#triage/needs-information">`triage/needs-information`</a> | Indicates an issue needs more information in order to work on it.| anyone | |
 | <a id="triage/not-reproducible" href="#triage/not-reproducible">`triage/not-reproducible`</a> | Indicates an issue can not be reproduced as described.| anyone | |
 | <a id="triage/unresolved" href="#triage/unresolved">`triage/unresolved`</a> | Indicates an issue that can not or will not be resolved.| anyone | |
+| <a id="wg/ipam" href="#wg/ipam">`wg/ipam`</a> | | anyone |  [label](https://prow.ci.kubevirt.io/command-help#label) |
```

Comment:
In this comment, please sync to sig-network charter in community repo. In sig network charter, please add wg-ipam.

Reply:
updated this PR (hope i understood correct)
and created https://github.com/kubevirt/community/pull/300